pageserver/src/page_cache.rs

```diff
@@ -95,9 +96,12 @@ impl AddAssign for PageCacheStats {
             num_page_images: self.num_page_images + other.num_page_images,
             num_wal_records: self.num_wal_records + other.num_wal_records,
             num_getpage_requests: self.num_getpage_requests + other.num_getpage_requests,
-            first_valid_lsn: self.first_valid_lsn + other.first_valid_lsn,
-            last_valid_lsn: self.last_valid_lsn + other.last_valid_lsn,
-            last_record_lsn: self.last_record_lsn + other.last_record_lsn,
+
+            // FIXME: needs review
+            // What should be happening here? I'm not sure what is the desired result.
+            first_valid_lsn: min(self.first_valid_lsn, other.first_valid_lsn),
+            last_valid_lsn: max(self.last_valid_lsn, other.last_valid_lsn),
+            last_record_lsn: max(self.last_record_lsn, other.last_record_lsn),
```

Comment:
I think I need advice here. I'm pretty sure that adding LSNs doesn't make sense, but is the min/max stuff here any better, or should the 3 LSNs just be removed from `PageCacheStas` altogether?

Reply:
I think @hlinnaka can tell more about it: is there any sense in calculating min/max LSNs for different timelines?
There is definitely no sense in comparing LSNs from different tenants and I hope we will resurrect multi-tenancy support sometimes.

Reply:
No, it doesn't really make sense. We can just remove them from the stats, at least from the aggregated stats. It would be nice to look at the LSNs of a specific timeline, but we don't have an UI for that so that that's academic at the moment.